pkg/kudoctl/util/vars/vars.go

```diff
@@ -11,4 +11,12 @@ var (
 	PackageVersion  string
 	StorageBucket   string
 	StoragePrefix   string
+
+	//FrameworkImportPath defines the location of the helm or KUDO framework definition that should be imported
+	FrameworkImportPath string
```

Comment:
vars are gone on master (I'll be merging today). You'll have to refactor this. Sorry :) they are not coming back on my watch! :D

Reply:
I fixed this